### PR TITLE
Feature/yonatanari/wellcome page responsive without media

### PR DESCRIPTION
**Pull request by 3dyonic**
https://monday.monday.com/boards/2861766393/pulses/2889507263

#### Basic
- [ ] Used plop (`npm run plop`) to create a new component.
- [ ] PR has description.
- [ ] New component is functional and uses Hooks. 
- [ ] Component defines [`PropTypes`](https://reactjs.org/docs/typechecking-with-proptypes.html).
#### Style
- [ ] Styles are added to `NewComponent.modules.scss` file inside of the `NewComponent` folder.
- [ ] Component uses CSS Modules.
- [ ] Design is compatible with [Monday Design System](https://design.monday.com/).
#### Storybook
- [ ] Stories were added to `/src/NewComponent/__stories__/NewComponent.stories.js` file.
- [ ] Stories include all flows of using the component.
#### Tests
- [ ] Tests are compliant with [TESTING_README.md](TESTING_README.md) instructions.

### CODE REVIEW

**Review by SergeyRoyt — APPROVED** [review]
Great PR!

File: src/storybook/stand-alone-documentaion/welcome/abilities-description/abilities-description.scss [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
 .monday-storybook-abilities-description {
   margin-top: $spacing-between-section-items;
-  width: 100%;
-  display: grid;
-  grid-template-columns: 1fr 1fr 1fr;
-  grid-gap: $spacing-large;
-}
\ No newline at end of file
+
+  @include grid-auto-fit($grid-gap: $spacing-large, $grid-cell-min-width: $abilities-cell-min-width);
```
Comment (hadasfa) [inline src/storybook/stand-alone-documentaion/welcome/abilities-description/abilities-description.scss:9]: cool!

File: src/styles/global-css-settings.scss [hunk 2]
```diff
@@ -44,3 +44,23 @@ $expand-animation-timing: cubic-bezier(0, 0, 0.35, 1);
...
+// -- grid adaptation is due to the min value along with auto-fit and 1fr max value.
+
+// @params:
+// - $grid-gap: null , defines grid "gap" attribute.
+// - $grid-column-gap: null,  defines grid "column-gap" attribute.
+// - $grid-row-gap: null,  defines grid "row-gap" attribute.
+// - $grid-cell-min-width: {mandatory}, defined min value in for grid-template-columns
+
+@mixin grid-auto-fit($grid-gap: null, $grid-column-gap: null, $grid-row-gap: null, $grid-cell-min-width) {
+  display: grid;
```
Comment (hadasfa) [inline src/styles/global-css-settings.scss:61]: very cool

**Review by hadasfa — APPROVED** [review]
Awesome!

**Pull request merged**